### PR TITLE
Adding in video, fixing title on English version

### PR DESCRIPTION
**Pull request by nomadscientist**

### CODE REVIEW

File: events/spanscriptomics/index-en.md [hunk 1]
```diff
@@ -32,8 +31,6 @@ slack_channel: "https://gtnsmrgsbord.slack.com/archives/C02HEN1RMA8"
 
 ---
 
-# Spanscriptomics: Single cell analysis using Galaxy
```
Comment (hexylena) [inline events/spanscriptomics/index-en.md:35]: You've deleted this from english but not spanish, is that correct?

File: events/spanscriptomics/index.md [hunk 2]
```diff
@@ -79,6 +76,8 @@ Por favor, deja tu dirección de correo electrónico para notificarte cuando el
 
 ## Spanscriptomics: "¿Por qué?"
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YE_QLodIe_w" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (hexylena) [inline events/spanscriptomics/index.md:79]: could you please use the nocookie version? just change `youtube` to `youtube-nocookie`

Comment (hexylena) [reply events/spanscriptomics/index.md:79]: you've also only added this to spanish, why not on the english version too?

Comment (nomadscientist) [reply events/spanscriptomics/index.md]: Really, it's in the link? Wow

File: events/spanscriptomics/index-en.md (hunk 1 above)
Comment (nomadscientist) [reply events/spanscriptomics/index-en.md:35]: It appears twice in the English one, hence removing it makes it appear once (hopefully)

Comment (hexylena) [reply events/spanscriptomics/index-en.md:35]: Ahhh ok I see what you mean. But I think you removed the wrong one, I think you need to leave this one, and remove the one below "En Español"

Comment (nomadscientist) [reply events/spanscriptomics/index-en.md:35]: Doh

Comment (nomadscientist) [reply events/spanscriptomics/index-en.md:35]: Done

Comment (hexylena) [reply events/spanscriptomics/index-en.md:35]: did you not push it yet?

Comment (nomadscientist) [reply events/spanscriptomics/index-en.md:35]: Shush....

Comment (nomadscientist) [reply events/spanscriptomics/index-en.md:35]: I did now...

**Comment by nomadscientist** [comment]
Can I get a quick merge of this and then we keep working on it? I have the Spanish programme links updated (ish, my brain is a bit mush), but my thoughts are, if the Spanish programme at least has the right links to tutorials/videos, it's a lot easier to send to the trainers to test all the tutorials.
Then we can keep working on getting the English one sorted and then can do the translations.

File: _data/materials.yaml [hunk 3]
```diff
@@ -1257,8 +1257,8 @@ introduction/galaxy-intro-short/slides/es:
   - captions: []
     event_name: GTÑ
     length: TODO
-    link: TODO # https://training.galaxyproject.org/training-material/videos/watch.html?v=introduction/tutorials/galaxy-intro-short/slides_ES
-    speaker: awspolly
+    link: https://training.galaxyproject.org/training-material/videos/watch.html?v=introduction/tutorials/galaxy-intro-short/slides_ES
+    speaker: Lupe
```
Comment (hexylena) [inline _data/materials.yaml:1261]: Lupe is not in our instructors list, awspolly is the correct value here

**Comment by hexylena** [comment]
> Can I get a quick merge of this and then we keep working on it? 

yeah I can fix the data errors and merge it.

**Comment by hexylena** [comment]
Oh, I can't, because you've opened the PR from your `main` branch, please avoid this if possible, create a branch first before making your PR.

Ok, I will merge it and fix it immediately then.

File: _data/materials.yaml [hunk 4]
```diff
@@ -1257,8 +1257,8 @@ introduction/galaxy-intro-short/slides/es:
   - captions: []
     event_name: GTÑ
     length: TODO
-    link: TODO # https://training.galaxyproject.org/training-material/videos/watch.html?v=introduction/tutorials/galaxy-intro-short/slides_ES
-    speaker: awspolly
+    link: https://training.galaxyproject.org/training-material/videos/watch.html?v=introduction/tutorials/galaxy-intro-short/slides_ES
```
Comment (hexylena) [inline _data/materials.yaml:1260]: this link is wrong, it needs to point to a youtube video, this will not work.

File: _data/materials.yaml [hunk 5]
```diff
@@ -1293,17 +1293,17 @@ transcriptomics/scrna-intro/slides/es:
...
   type: Lecture
   tags: [transcriptomics, single-cell]
   videos:
   - captions:
     - mtekman
-    event_name: Smörgåsbord
+    event_name: GTÑ
     length: 20M
-    link: D3qvYWZMFa0
+    link: https://training.galaxyproject.org/training-material/videos/watch.html?v=transcriptomics/tutorials/scrna-intro/slides_ES
```
Comment (hexylena) [inline _data/materials.yaml:1306]: same issue here, will not work, must be a youtube video. I'll get those uploaded today and revert this for now.

**Pull request merged**